### PR TITLE
change record (crischutu07.is-a.dev)

### PR DESCRIPTION
**Pull request by crischutu07**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://crischutu07.github.io

### CODE REVIEW

**Comment by 21Z** [comment]
Are you sure you want to use `URL` and not `CNAME`?

File: domains/crischutu07.json [hunk 1]
```diff
@@ -4,6 +4,6 @@
         "email": "sondolavnxbox.omlet@gmail.com"
     },
     "record": {
-        "A": ["159.138.147.9"]
+        "URL":"crischutu07.github.io"
```
Comment (CuteDog5695) [inline domains/crischutu07.json:7]: doesnt have protocol

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by CuteDog5695** [comment]
> Welp, I guess sticking with CNAME

if you want to stick to CNAME then replace "URL" with "CNAME"

**Review by iostpa — APPROVED** [review]
seems good to me now

**Pull request merged**